dgl_ptm/dgl_ptm/config.py

```diff
@@ -33,6 +33,8 @@ def _convert_parameters(cls, v):
     model_config = ConfigDict(validate_default = True)
 
 
+# TODO(tvl): describe the difference between SteeringParams and Config.
+# TODO(tvl): there is some overlap between the parameters defined in SteeringParams and Config; this should be resolved. Otherwise, it just causes confusion.
```

Comment:
This info is hidden in the codes here. I suggest submitting an issue for this.

Reply:
Resolved this in new commits in this pull request: added short description to config and steering parameters and removed duplicate parameters.